tests/Unit/PointwiseFunctions/AnalyticData/Xcts/Test_Binary.cpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <optional>
+#include <string>
+
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "PointwiseFunctions/AnalyticData/Xcts/Binary.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace Xcts::AnalyticData {
+namespace {
+
+using test_tags = tmpl::list<
+    Tags::ConformalMetric<DataVector, 3, Frame::Inertial>,
+    Tags::InverseConformalMetric<DataVector, 3, Frame::Inertial>,
+    ::Tags::deriv<Tags::ConformalMetric<DataVector, 3, Frame::Inertial>,
+                  tmpl::size_t<3>, Frame::Inertial>,
+    gr::Tags::TraceExtrinsicCurvature<DataVector>,
+    Tags::ShiftBackground<DataVector, 3, Frame::Inertial>,
+    Tags::LongitudinalShiftBackgroundMinusDtConformalMetric<DataVector, 3,
+                                                            Frame::Inertial>,
+    Tags::ConformalFactor<DataVector>>;
+
+template <typename IsolatedObjectRegistrars>
+struct BinaryProxy {
```

Comment:
I think it's probably best to avoid using the term `proxy` outside of referring to the charm construct.

Reply:
Agreed, though this "proxy" business is done in all other analytic solution tests (for all systems) so I think it's better to be consistent in this PR. But would be good to rename them.